devops/azure-pipelines/dockerize-template.yml

```diff
@@ -0,0 +1,21 @@
+parameters:
+  - name: 'jobName'
+    type: string
+  - name: 'containerRegistry'
+    type: string
+  - name: 'repository'
+    type: string
+  - name: 'dockerPath'
+    type: string
```

Comment:
document the parameters (i.e. containerRegistry is supposed to be the name of a service connection, and dockerPath the path to a dockerfile. but that's not obvious from their names)